ecl/agentexec/agentexec.cpp

```diff
@@ -284,14 +284,18 @@ class WaitThread : public CInterfaceOf<IPooledThread>
                     std::list<std::pair<std::string, std::string>> params = { };
                     if (compConfig->getPropBool("@useThorQueue", true))
                         params.push_back({ "queue", queue.get() });
-                    StringBuffer jobName(wuid);
                     if (isThorAgent)
                     {
-                        jobName.append('-').append(graphName);
                         params.push_back({ "graphName", graphName.get() });
                         params.push_back({ "wfid", std::to_string(wfid) });
                     }
 
+                    const char *thorName = compConfig->queryProp("@targetName");
```

Comment:
there won't always be a `@targetName` .. (only the Thor template currently sets it).
It would be better to keep the old route, and make the new naming condtional, e.g.:
```
                    const char *targetName = compConfig->queryProp("@targetName");
                    StringBuffer jobName;
                    if (targetName)
                    {
                        jobName.append(targetName).append('-');
                        jobName.append(myInstanceNum);
                    }
                    else
                        jobName.append(wuid).append('-').append(graphName);
```

Reply:
See comments on myInstanceNum elsewhere.